### PR TITLE
EHPR-74 | Creation of standalone package for api

### PR DESCRIPTION
**Pull request by fwpushan**
Build script to create api artifact for serverless deployment

### CODE REVIEW

File: packing-api.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# Script to pack api app for severless or micro contanier service
+echo 'Building ... \n' && yarn workspace @ehpr/api build && \
+echo 'Updating prod dependencies...\n' && yarn workspaces focus @ehpr/api --production  && \
+echo 'Deleting existing build dir\n' && rm -rf ./.build || true && \
+echo 'Creating build dir...\n' && mkdir -p .build/api && \
+echo 'Copy Node modules....\n' && cp -r node_modules .build/api && \
```
Comment (arranfw) [inline packing-api.sh:7]: Q: Does this rely on an empty node_modules at the start of the script?

File: packing-api.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+echo 'Unlink local packages...\n' && rm -rf .build/api/node_modules/@ehpr/* && \
+echo 'Hardlink local packages...\n' && cp -r ./packages/* .build/api/node_modules/@ehpr/ && \
```
Comment (arranfw) [inline packing-api.sh:9]: Are packages already built implicitly above?

**Review by arranfw — APPROVED** [review]

File: packing-api.sh (hunk 1 above)
Comment (fwpushan) [reply packing-api.sh:7]: `yarn workspaces focus @ehpr/api --production` create node_modules for sure

File: packing-api.sh (hunk 2 above)
Comment (fwpushan) [reply packing-api.sh:9]: dependent packages ...

**Review by arranfw — APPROVED** [review]

File: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+/*!
```
Comment (dbayly-freshworks) [inline .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs:8]: Is this code here meant to be human readable? We might want to break it up a bit.

**Review by dbayly-freshworks — APPROVED** [review]
One nit, otherwise looks good

File: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs (hunk 3 above)
Comment (arranfw) [reply .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs:8]: I think this is the module cache

File: apps/api/src/app.module.ts [hunk 4]
```diff
@@ -1,10 +1,10 @@
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
-import { FormModule } from './form/form.module';
+// import { FormModule } from './form/form.module';
```
Comment (hitankar-freshworks) [inline apps/api/src/app.module.ts:4]: Why is this commented?

File: packing-api.sh [hunk 5]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# Script to pack api app for severless or micro contanier service
+echo 'Building ... \n' && yarn workspace @ehpr/api build && \
```
Comment (hitankar-freshworks) [inline packing-api.sh:3]: Might be a good idea to clear the previous  zip file.

File: apps/api/src/app.module.ts (hunk 4 above)
Comment (fwpushan) [reply apps/api/src/app.module.ts:4]: Try to fix crash.... some WIP code merged into main branch.

File: packing-api.sh (hunk 5 above)
Comment (fwpushan) [reply packing-api.sh:3]: Zip is cleaned in following steps

**Review by hitankar-freshworks — APPROVED** [review]

**Pull request merged**